xbmc/dialogs/GUIDialogSmartPlaylistEditor.cpp

```diff
@@ -546,6 +549,37 @@ void CGUIDialogSmartPlaylistEditor::HighlightItem(int item)
   OnMessage(msg);
 }
 
+std::vector<CGUIDialogSmartPlaylistEditor::PLAYLIST_TYPE> CGUIDialogSmartPlaylistEditor::GetAllowedTypes(const std::string& mode)
+{
```

Comment:
Should I move this method somewhere else? The method is static and perhaps better belongs to CSmartPlaylist?